lambdas/hello-world/serverless.yml

```diff
@@ -0,0 +1,14 @@
+HelloWorld:
+  handler: hello-world.handler
+  name: ${self:service.name}-hello-world-lambda
```

Comment:
where is ${self:service.name} comming from?

Reply:
The main file (unfortunately, makes it a bit harder to read like this as you see...)